react-backend/client/src/App.js

```diff
@@ -138,7 +120,7 @@ class App extends Component {
                 )
               }
             />
-
+            <Route exact path="/internalServerError" component={InternalServerError} />
```

Comment:
you'll need to add this to `frontEndRoutes.js`

Reply:
Done!